### ISSUE TITLE
[Design] Low contrast on some Tooltips/Buttons

### ISSUE THREAD

**Issue by Dr1Ku** [keybase/keybase-issues#1490]
Just a small reminder that contrast on the "actions" tooltip (when logged in), displaying labels such as "Sign", "Verify" and the Submit button on the Account Creation form is pretty low. 

I spent some 2-3 minutes trying to figure out why my 12+ chars password wasn't being accepted.

![keybase-readability](https://cloud.githubusercontent.com/assets/373565/6958625/f199b6b8-d90e-11e4-92f6-78e494ffac6d.png)

http://contrastrebellion.com/ ! :+1:

**Comment by jackmawer** [comment]
Indeed, when verifying, it's VERY tricky to read the bar telling me my password must be longer!
